### PR TITLE
Resolved start script as per #26

### PR DESCRIPTION
**Pull request by tjeaneric**
#### What does this PR do?
* Fixes issue #26 

#### Description of Task to be completed?
* use `npm run dev` to run the development script
* use `npm start` to run the production script

#### How should this be manually tested?
* `git pull` 
* run git checkout to this branch and run `npm install` 
* run the production and development scripts separately

#### Any background context you want to provide?
1. I used Webpack `5.73.0` and webpack-merge  `5.8.0`

#### What are the relevant pivotal tracker stories?
*  We need to create a script to run the application in development and adjust the start script to run the built application in production mode.

#### Questions:
No.

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -8,15 +8,22 @@ const port = process.env.PORT;
...
     path: path.resolve(__dirname, "dist"),
+    clean: true,
+    assetModuleFilename: "[name][ext]",
   },
+
   devServer: {
+    static: {
+      directory: path.resolve(__dirname, "dist"),
+    },
+    port: 3000,
```
Comment (Anguandia) [inline webpack.common.js]: Good and speedy work! I think we can read the port from the environment. What if I also have my local backend running on 3000 by default? Please revert to reading from the environment, you can reserve the 3000 as a default/fallback port though

**Review by Anguandia — CHANGES_REQUESTED** [review]
Good work, Just the port to the environment and serve the build in production

File: package.json [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-        "webpack-cli": "^4.9.2",
-        "webpack-dev-server": "^4.8.1"
-    }
+  "name": "dominators-phantom-fe",
+  "version": "1.0.0",
+  "description": "dream phantom project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack --config webpack.prod.js",
```
Comment (Anguandia) [inline package.json]: Also here, it looks like you are only building and not serving. Please try running in production mode on your machine and confirm

Comment (tjeaneric) [reply package.json]: Now serving production build.

File: webpack.common.js (hunk 1 above)
Comment (tjeaneric) [reply webpack.common.js]: Now port is read from environment

**Review by Anguandia — APPROVED** [review]

**Review by furebo — APPROVED** [review]

**Pull request merged**